knack/config.py

```diff
@@ -111,12 +111,12 @@ def items(self, section):
         import re
         pattern = self.env_var_name(section, '.+')
         candidates = [(k.split('_')[-1], os.environ[k], k) for k in os.environ if re.match(pattern, k)]
-        result = {c[0]: c for c in candidates}
+        result = {c[0].lower(): c for c in candidates}
         for config in self._config_file_chain if self.use_local_config else self._config_file_chain[-1:]:
             try:
                 entries = config.items(section)
                 for name, value in entries:
-                    if name not in result:
+                    if name.lower() not in result:
```

Comment:
We should force lower case for all configs in files, otherwise we need to change `get` and `set` as well.

Reply:
Shall we add force rule in `set_value`?